exercises/concept/monster-attack/.docs/introduction.md

```diff
@@ -1 +1,146 @@
-Will be generated from introduction.md.tpl
+# Introduction
```

Comment:
Also, it seems you didn't shorten the introduction and removed the links...

Reply:
Hmmm, maybe I don't understand how the .tpl file and generation stuff works. I was expecting this to happen post commit. Is there a configlet thing to do?

Reply:
And I'm not sue I understood what you meant about the links and the shortening. I usually keep introduction and about the same, if there is a reason not to do this, I still don't understand it! I think I have seen an explanation written down before, but it didn't seem to make enough sense for me to want to do it ...

Reply:
> Hmmm, maybe I don't understand how the .tpl file and generation stuff works. I was expecting this to happen post commit. Is there a configlet thing to do?

Right. it's not automatic, you still have to run `bin/configlet generate`.

> And I'm not sue I understood what you meant about the links and the shortening. I usually keep introduction and about the same, if there is a reason not to do this, I still don't understand it! I think I have seen an explanation written down before, but it didn't seem to make enough sense for me to want to do it ...

It's just [the specs](https://exercism.org/docs/building/tracks/concepts) for introduction.md:

> This file is shown if a student has not yet completed the corresponding concept exercise. It should provide a brief introduction to the concept.
>
> -    Only information that is needed to understand the fundamentals of the concept should be provided. Extra information should be left for the about.md document.
> -    Links should be used sparingly, if at all. While a link explaining a complex topic like recursion might be useful, for most concepts the links will provide more information than needed so explaining things concisely inline should be the aim.
> -    Proper technical terms should be used so that the student can easily search for more information.
> -    Code examples should only be used to introduce new syntax (students should not need to search the web for examples of syntax). In other cases provide descriptions or links instead of code.
> -    Don't refer to an exercise in the text, as this file is displayed outside the context of an exercise.

In a nutshell, it should be short and to the point. Links and details are distracting.